### ISSUE TITLE
Report--Success--Builds-Kiosk-Variant-without-errors---Ubuntu-16.04.4-amd64---AMD---Intel

### ISSUE THREAD

**Issue by claudiusraphaelpaeth** [DebianDog/MakeLive#2]
.. just to report that i tested building using Kiosk  option based dog-boot-stretch-20180224 64-bit;

Hardware used to build and test:

- Dell OptiPlex 740 Enhanced
[  AMD Phenom Quad 9600B | ATI FirePro V5700 | 6.5 GiB DDR II Async | Samsung HD160J  ]
.. time taken ~ 20-25 minutes in background, while doing daily operation;

- HP EliteBook 2740p
[ Intel Core i5 Quad vPro | Intel Ironlake | 8 GiB DDR III Dual | Intel 1.8" µSATA SSD 80G ]
.. time taken ~ 12 to 15 minutes, no other applications in use;

OS used to build:
Ubuntu Xenial Xerus
[ Canonical Ubuntu Linux 16.04.4 64-Bit Unity Xorg X11- Fresh and standard installations ]

Building succeeded on all 3 Variants of Xenial Root on
[ A: EXT4 | B: BTRFS | C: ZFS ]

Besides "yad", which was missing and had to be installed forefront, no issues to report in any given constellation.

Manually tested given functionality without any issues to report in any built on both systems.

To sum it up:

Works well!  [ .. at least for the Kiosk Variant; for now. ]
